.github/workflows/ci.yml

```diff
@@ -21,12 +21,13 @@ jobs:
       fail-fast: false
       matrix:
         BACKEND: [memory, cassandra, scylladb, hbase, rocksdb]
+        java-version: ['8', '8.0.192']
```

Comment:
since one latest version is ok and can remove the '8.0.192'.
and also keep JAVA_VERSION style?

Reply:
Cool, I updated the workflow. 👍🏼